src/infer.py

```diff
@@ -47,7 +48,7 @@ def infer(gen, num_fonts, path, style_dim=100, resolution=64, device='cuda'):
     # Generate a batch of fake characters
     fake_chars = gen(char_vector, style_vector)
     fake_chars = torch.reshape(fake_chars, [num_fonts, -1, 2])
-    rasters = rasterize(fake_chars)
+    rasters = rasterizer.Rasterizer()(fake_chars)
```

Comment:
you don't use the resolution argument to the infer func. intentional?

Reply:
No... not done with this PR yet actually.

Reply:
Yeah I noticed the draft.